skbase/lookup/_lookup.py

```diff
@@ -190,34 +190,23 @@ def _filter_by_tags(obj, tag_filter=None, as_dataframe=True):
     if tag_filter is None:
         return True
 
-    type_msg = (
```

Comment:
it looks like the function already allowed for iterable of str or str

Reply:
it seems like you are still changing the logic itself, making it less general, and removing the useful error message.